src/lib/components/pages/products/product-pricing-pannel.svelte

```diff
@@ -64,6 +66,8 @@
 			});
 			return;
 		}
+    isLoadingStore.set(true);
```

Comment:
scope inside component, don't use store like this.
That's why svelte has $state feature for it.
use $state instead